Mathlib/CategoryTheory/GradedObject.lean

```diff
@@ -80,6 +85,39 @@ def eval {β : Type w} (b : β) : GradedObject β C ⥤ C where
 
 section
 
+variable {β : Type*} (X Y : GradedObject β C)
+
+/-- Constructor for isomorphisms in `GradedObject` -/
+@[simps]
+def isoMk (e : ∀ i, X i ≅ Y i) : X ≅ Y where
+  hom i := (e i).hom
+  inv i := (e i).inv
+
+variable {X Y}
+
+lemma isIso_of_isIso_apply (f : X ⟶ Y) [hf : ∀ i, IsIso (f i)] :
+    IsIso f := by
```

Comment:
Should this be an `instance`?

Reply:
I am not sure how to accurately test that it does create a loop, but I do not think it should be an instance, similarly as `NatIso.isIso_of_isIso_app` is not an instance. I have added a comment.